src/main/java/edu/ucsb/cs156/dining/controllers/ReviewController.java

```
@@ -0,0 +1,91 @@
+@RequestMapping("/api/reviews")
+@RestController
+@Slf4j
+public class ReviewController extends ApiController {
+
+    @Autowired
+    ReviewRepository reviewRepository;
+
+
+    /**
+     * This method returns a list of all Reviews.
+     * 
+     * @return a list of all Reviews
+     */
+    @Operation(summary = "List all Reviews")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/all")
+    public Iterable<Review> allReviews() {
+        log.info("Attempting to log all reviews");
+        Iterable<Review> reviews = reviewRepository.findAll();
+        log.info("all reviews found, ", reviews);
+        return reviews;
+    }
+
+    /**
+     * This method allows a user to submit a review
+     * 
+     * @return message that says an review was added to the database
+     * @param itemId         id of the item
+     * @param dateItemServed localDataTime
+     *                       All others params must not be parameters and instead
+     *                       derived from data sources that are dynamic (Date), or
+     *                       set to be null or some other signifier
+     */
+    @Operation(summary = "Create a new review")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @PostMapping("/post")
+    public Review postReview(
+        @Parameter(name = "itemId") @RequestParam String itemId,
+        @Parameter(name = "dateItemServed", description = "date (in iso format, e.g. YYYY-mm-ddTHH:MM:SS; see https://en.wikipedia.org/wiki/ISO_8601)") @RequestParam("dateItemServed") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) LocalDateTime dateItemServed) // For an explanation of @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME) See: https://www.baeldung.com/spring-date-parameters
+
+        throws JsonProcessingException {
+        LocalDateTime now = LocalDateTime.now();
+        Review review = new Review();
+        review.setDateItemServed(dateItemServed);
+        review.setDateCreated(now);
+        review.setItemId(itemId);
```

You need to look up the itemId in the MenuItems table (which needs to exist before this PR can be merged).  And if that itemId doesn't exist in the MenuItems table, you need to throw an exception.

Similar to this code:
https://github.com/ucsb-cs156/proj-courses/blob/4bdc2674508428d03cb2dcd2e9077d75cdaf5a3f/src/main/java/edu/ucsb/cs156/courses/controllers/PSCourseController.java#L114